po/R-zh_CN.po

```diff
@@ -165,61 +170,118 @@ msgstr "第一个元素应为下界；"
 msgid "the second element should be the upper bound(s)."
 msgstr "第二个元素应为上界。"
 
-msgid "x."
-msgstr "x."
+msgid "forderv(query) took ..."
+msgstr "forderv(query) 用了 ..."
+
+msgid "Generating final logical vector ..."
+msgstr "产生最后的逻辑向量 ..."
 
-msgid "is type"
-msgstr "的类型为"
+msgid "done in"
+msgstr "结束在"
```

Comment:
msgstr "花费了"

Since "done in" is always followed by the time the function it costs, I suggested translating it as "it took" (花费了多少时间)。 Let me know how you think of it.